views/lib/loggedin.hbs

```diff
@@ -74,6 +74,11 @@
                 <i class="fa fa-bars"></i>
             </a>
             <div class="content-min-height">
+                <section class="demo" id="offlineAlert" style="display:none;">
+                    <div class="alert alert-warning" role="alert">
+                        <p>Du bist offline. <span class="hidden-md-down">Um dir trotzdem etwas anzuzeigen siehst du eventuell veraltete Inhalte.</span></p>
```

Comment:
maybe we can also show here how old the info is?

Reply:
the offline alert should be enough here. to show a timestamp of the response shown the calendar script must be changed in how to load the events.